weatherman.py

```diff
@@ -0,0 +1,189 @@
+import calendar
+import csv
+import os
+import sys
+
+
+class temperatureTracker():
+
+    def __init__(self):       
+        self.max_temperature = float('-inf')
+        self.max_temperature_date = ""
+        self.min_temperature = float('inf')
+        self.min_temperature_date = ""
+        self.max_humidity = float('-inf')
+        self.max_humidity_date = ""
```

Comment:
Any specific reason of defining them in constructor? Can we define them on the go as needed? Look what is recommended in Python and why.

Reply:
Still unaddressed?